frappe/templates/emails/auto_email_report.html

```diff
@@ -1,11 +1,23 @@
 {% macro get_alignment(col) %}
 {%- if col.fieldtype in ('Int', 'Float', 'Currency', 'Check') %} class="text-right" {% endif -%}
 {% endmacro %}
+
+{% macro format(col, row) %}
+{% if col.fieldtype == 'Currency' %}
+        {% if row[col.fieldname] != '' %}
+                 {{ frappe.utils.fmt_money(row[col.fieldname], currency=frappe.db.get_value("Global Defaults", None, "default_currency"), precision=2) }}
```

Comment:
You may also need to check if a row has the currency column and if yes format based on that column currency value...

Reply:
As I understand the code in auto_rmail_report.py it is not possible for a row fieldtype to be different from a column setting (get_report only returns 2 vals, a column list and a rows list).

Reply:
@RJPvT I checked the codebase and it seems like `frappe.format` already handles currency formatting. What's the reason for these changes?

Reply:
@surajshetty3416 in the jinja enviroment the frappe.format call cant know of the fieldtype. So it correctly finds out if its a 'number' field but cant distinguis between monitairy and normal float. This fix repears that. I"ve treid fixing thd python side but that keeps sinding me unicode symbols that het html render botches.